packages/utilities/psammead-styles/src/font-styles.js

```diff
@@ -62,3 +78,19 @@ export const getSerifBold = service => {
   const { serifBold } = fonts[service];
   return serifBold || getSansBold(service);
 };
+
+export const getSerifLight = service => {
+  if (!fonts[service]) {
+    return null;
+  }
+  const { serifLight } = fonts[service];
+  return serifLight || getSansRegular(service);
+};
+
+export const getSerifLightItalic = service => {
+  if (!fonts[service]) {
+    return null;
+  }
+  const { serifLightItalic } = fonts[service];
+  return serifLightItalic || getSansLight(service);
```

Comment:
```suggestion
  return serifLightItalic || getSerifLight(service);
```

Reply:
Removed since it isn't used at the moment so I think it would be better to implement once it is used and UX has confirmed what it should fall back to.